_includes/pbc/all/install-features/202304.0/install-the-warehouse-picking-shipment-feature.md

```diff
@@ -14,8 +14,8 @@ To start feature integration, integrate the required features:
 
 | NAME              | VERSION          | INTEGRATION GUIDE                                                                                                                              |
 |-------------------|------------------|------------------------------------------------------------------------------------------------------------------------------------------------|
-| Warehouse Picking | {{site.version}} | [Warehouse Picking feature integration](/docs/scos/dev/feature-integration-guides/{{site.version}}/install-the-warehouse-picking-feature.html) |
-| Shipment          | {{site.version}} | [Shipment feature integration](/docs/pbc/all/carrier-management/{{page.version}}/install-and-upgrade/install-the-shipment-feature.html)        |
+| Warehouse Picking | {{site.version}} | [Warehouse Picking feature integration](/docs/scos/dev/feature-integration-guides/202304.0/install-the-warehouse-picking-feature.html) |
```

Comment:
We should not hardcode the version number. Instead {{site.version}} or {{page.version}} variables are to be used. See the difference here: https://docs.spryker.com/docs/scos/user/intro-to-spryker/contribute-to-the-documentation/style-formatting-general-rules.html#working-with-versions Try using {{page.version}} here